### PR TITLE
Noto Serif Khitan Small Script: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by simoncozens**
f1b788b: [gftools-packager] Noto Serif Khitan Small Script: Version 1.000; ttfautohint (v1.8.4.7-5d5b) added

* Noto Serif Khitan Small Script Version 1.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/khitan-small-script.git at commit https://github.com/notofonts/khitan-small-script/commit/.

### CODE REVIEW

**Comment by RosaWagner** [comment]
@simoncozens do you know if with the last merge of glyphsets, this language/subsets/primary_script could be fixed in this PR?

**Comment by simoncozens** [comment]
Yes, glyphsets now has this. We don't have a language for it, because it's another undeciphered script but I've put a sample text (taken from the [Kaesong mirror inscription](https://wikisource.org/wiki/Kaesong_mirror_inscription)) in the METADATA.pb.

**Comment by RosaWagner** [comment]
@simoncozens the linespacing is a bit intense, no? Shouldn't it follow the CJK linespacing rules?

Screenshot 2023-06-22 at 13 43 43

**Comment by simoncozens** [comment]
This script is very different to other CJK scripts because the characters stack vertically:

shape

**Comment by RosaWagner** [comment]
Okay makes sense. thanks!

File: ofl/notoserifkhitansmallscript/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/khitan-small-script.git"
+  archive_url: "https://github.com/notofonts/khitan-small-script/releases/download/NotoSerifKhitanSmallScript-v1.000/NotoSerifKhitanSmallScript-v1.000.zip"
+}
+is_noto: true
+sample_text {
+  masthead_full: "𘭔𖿤𘭢 𘲽𘱸"
+  masthead_partial: "𘰷𘭴"
+  styles: "𘰷𘭴 𘰝𘲥𘲦 𘲲𘯴𘱆 𘳍𘰄 𘰴𘳍𘰄 𘬜𘭞𘰷 𘬜𘭞𘰷 𘰷𘭞𘭟 𘰷𘭞𘭟 𘱓𘮯𘮡"
```
Comment (chrissimpkins) [inline ofl/notoserifkhitansmallscript/METADATA.pb:27]: Fonts eng requested that I edit the length of this string to address the very tall appearance of this family on the front end.  Will open a new PR with these changes.

**Pull request merged**